test/sql/function/string/test_string_split.test

```diff
@@ -0,0 +1,171 @@
+# name: test/sql/function/string/test_string_split.test
+# description: String split test
+# group: [string]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+SELECT string_split(string_agg(range, 'DUCK '), ' '), mod(range, 100) xx FROM range(50000) GROUP BY xx
+
+statement ok
+SELECT string_split(string_agg(range, '🦆 '), ' '), mod(range, 100) xx FROM range(50000) GROUP BY xx
```

Comment:
don't you want to do a result check with those queries as well?

Reply:
Sure, but then I'd prefer to lower the number and length of strings that are being split. Checking the result of these queries would mean a huge test file. Another option is to select COUNT(UNNEST(...)) and checking whether the strings are being split into the correct number of strings

Reply:
You can also add a LIMIT to your query

Reply:
Alternatively you can also check a result hash, see e.g. `test/sql/aggregate/aggregates/test_string_agg_big.test`. You can generate a result hash by adding `mode output_hash` into the test file, which will generate output hashes of any output the database generates. First make sure that output is correct though :)

Reply:
I like this hash idea!